src/lib/components/artist/articles/index.tsx

```diff
@@ -49,7 +51,13 @@ export default Relay.createContainer(Articles, {
 })
 
 interface RelayProps {
-  articles: Array<{
-    __id: string,
-  } | null> | null,
+  articles:
+    | Array<
+
+        | {
+          __id: string,
+        }
+        | null
+    >
+    | null
```

Comment:
It's actually not a bug, but clearly doesn't look good.

Because you added a `\n` inside of the object, it expands it. And the expanded form of `|` is to put a leading `|`.

It hasn't been an issue with flow because it has `?` operator. But given that typescript relies heavily on `| null` or `| undefined`, it triggers way more.

I need to add a special case for it. Thanks for the report!